### ISSUE TITLE
macro error

### ISSUE THREAD

**Issue by cbussi** [ScienceToolkit/MiNA#1]
Hi, I have tried to run both versions of MiNA but after clicking run I got this macro error message:

statement cannot beging with '<' in line 7.
!DOCTYPE html

Thank you!

**Comment by ScienceToolkit** [comment]
Hello,

Version 2 is not completed yet and will not run. Version 1 should work, I have tested it on a few images and it works for me. 

I am going to rename the two folders and push the most current files I have. I have updated version one's folder to contain the original workflow in MiNA_V100.ijm and an updated workflow using a different skeletonizing routine in MiNA_V110.ijm. I think it may have just been how you had installed the macro that was giving you problems. 

_Please follow the steps below and let me know if it works or if you are still having issues._

1. Download the repository as a zip file (Using __Clone or Download__ on MiNA's main GitHub page). 
2. Extract the version 1 macro, __MiNA_110.ijm__ found in the __/version_1__ directory, to a folder on your computer.  
3. Open ImageJ.
4. Install the macro by navigating to the file through Plugins ->Macros -> Install... and select the MiNA_110.ijm file. 
5. Try to run it again and let me know if you get the same error as before. 

Cheers,
Andrew
